src/entities/postgres-storage/db-shelley.js

```diff
@@ -166,54 +166,74 @@ class DBShelley extends DB<TxType> implements Database<TxType> {
     await conn.query(sql)
   }
 
+  async storeGroupAddresses(groupAddresses: Array<any>): Promise<void> {
+    const sql = Q.sql.insert().into('group_addresses')
+      .setFieldsRows(groupAddresses.map(addr => ({
+        group_address: addr.groupAddress,
+        utxo_address: addr.utxoAddress,
+        account_address: addr.accountAddress,
+      })))
+      .onConflict()
+      .toString()
+    this.logger.debug('storeGroupAddresses: ', sql)
+    await this.getConn().query(sql)
+  }
+
+  getGroupAddressesData(txDbData: TxDbDataType) {
+    const wasm = global.jschainlibs
+    const {
+      inputAddresses, outputAddresses,
+    } = txDbData
+    const allAddresses = _.uniq([...inputAddresses, ...outputAddresses])
+    this.logger.debug(`metadataCreator.allAddresses = ${allAddresses}`)
```

Comment:
This log doesn't make much sense anymore since we no longer have the `metadataCreator` callback.